dispatcher/hub/commonHub.go

```diff
@@ -22,6 +23,7 @@ type commonHub struct {
 	broadcast          chan data.BlockEvents
 	broadcastRevert    chan data.RevertBlock
 	broadcastFinalized chan data.FinalizedBlock
+	cancelFunc         func()
```

Comment:
suggestion to add here a close chan:
```GO
closeChan chan struct{}
```
init it in the constructor, call the 
```GO
close(wh.closeChan)
```
in the commonHub.Close function
and use it something like this:
```GO
func (wh *commonHub) Broadcast(events data.BlockEvents) {
    select{
        case wh.broadcast <- events:
        case <- wh.closeChan:
    }    
}
```
what do we gain? All external calls will not hang if we close this component

Reply:
thanks :+1:  added a close chan